File: GymFit/GymFit.Core/Entities/Notification.cs
namespace GymFit.Core
{
    public class Notification : BaseEntity
    {
        public string Content { get; set; }
        public bool Read { get; set; }
        public bool Deleted { get; set; }
        public DateTime? DateRead { get; set; }
        public DateTime? SendOnDate { get; set; }
        public DateTime? Seen { get; set; }

        public int UserId { get; set; }
        public User User { get; set; }
    }
}

File: SeminarskiGymFit/UI/gymfit_trainer/lib/models/notification.dart
import 'package:gymfit_trainer/models/user.dart';

class Notifications {
  late int id;
  late String content;
  late bool? Read;
  late DateTime? DateRead;
  late DateTime? SendOnDate;
  late DateTime ?Seen;
  late User? user;
  late int? userId;
  late bool isSelected = false;

Notifications({
  required this.id,
  required this.content,
  required this.Read,
 this.DateRead,
 this.Seen,
 this.SendOnDate,
 this.user,
 this.userId,
 required this.isSelected,
 
});

  Notifications.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    userId = json['userId'];
    content = json['content'];
    Read = json['read'];
    SendOnDate = DateTime.parse(json['sendOnDate']);
    user = User.fromJson(json['user']);
    
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['content'] = content;
    data['read'] = Read;
    data['sendOnDate'] = SendOnDate;
    return data;
  }



}

